command/tree.txt

```diff
@@ -0,0 +1,33 @@
+|-- admin           Perform admin activities with an indexer
```

Comment:
We should generate this from urfav command objects if we really want it.

Reply:
Agree, but did not find any built-in way to do it.